core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -235,7 +235,7 @@ open class TileInfo {
             for (unique in cityWideUniques + civWideUniques) {
                 val tileType = unique.params[1]
                 if (tileType == improvement) continue // This is added to the calculation in getImprovementStats. we don't want to add it twice
-                if (matchesUniqueFilter(tileType, observingCiv)
+                if (matchesTerrainFilter(tileType, observingCiv)
```

Comment:
[+1] Production from every [Mine] should be a valid civInfo unique, there's someone that already has something like that but I forget who

Reply:
That would be calculated in the `getImprovementStats()` function right? This function is only for getting the yield of the base tile